app/models/organisation.rb

```diff
@@ -34,6 +36,6 @@ def is_government?
   end
 
   def self.service_owner
-    Organisation.find_by(service_owner: true)
+    find_by(service_owner: true, iati_reference: SERVICE_OWNER_IATI_REFERENCE)
```

Comment:
Since there's a database index is on `iati_reference`, I'm wondering if it'd be quicker to use that column alone here?

Perhaps the Organisation factory could also use this constant?